content/kubermatic/master/architecture/requirements/support_policy/provider_support_matrix/azure/azure.en.md

```diff
@@ -47,22 +45,12 @@ Get your Tenant ID
 
 ```bash
 az account show --query tenantId -o json
-
-********-****-****-****-************
 ```
 
 create a new app with
 
 ```bash
 az ad sp create-for-rbac --role="Kubermatic" --scopes="/subscriptions/********-****-****-****-************"
-
-{
-  "appId": "********-****-****-****-************",
-  "displayName": "azure-cli-2018-11-25-08-01-39",
-  "name": "http://azure-cli-2018-11-25-08-01-39",
-  "password": "********-****-****-****-************",
-  "tenant": "********-****-****-****-************"
-}
```

Comment:
Is this really unwanted? I think it can be useful so users are aware of what output they should expect. I would eventually split it into two code blocks so it's easier to copy the command itself. @kron4eg what do you think?

Reply:
I found that is removed from other azure docs under kubeone.. so tried to keep it consistent was the intention here for cleanup. :)

Reply:
yeah, I think we can leave this. Maybe try to signal somehow that this is in fact an output of the command. But I'm not sure about that, it was OK for me.

Reply:
> Maybe try to signal somehow that this is in fact an output of the command.

I agree. As I mentioned above, we can split it into two code blocks, and add something like "The output should look like the following one:" to signalize it's the output.

Reply:
Taken care. Please review now!